api/CustomElementRegistry.json

```
@@ -113,6 +113,41 @@
             "deprecated": false
           }
         },
+        "connectedMoveCallback_lifecycle_callback": {
+          "__compat": {
+            "description": "Supports `connectedMoveCallback()` lifecycle callback",
```

Nit: Isn't the "Supports " prefix implied?

```suggestion
            "description": "`connectedMoveCallback()` lifecycle callback",
```